### DISCUSSION TITLE
关于加入 HEVC over FLV 的支持

### DISCUSSION THREAD

**Discussion by delhi2050** [zhongfly/mpv-build#73] [General]
[FFmpeg-Builds/patches/ffmpeg/release/6.0/0002-Support-for-HEVC-over-FLV.patch at master · yt-dlp/FFmpeg-Builds](https://github.com/yt-dlp/FFmpeg-Builds/blob/master/patches/ffmpeg/release/6.0/0002-Support-for-HEVC-over-FLV.patch)

**Comment by zhongfly** [comment]
我不打算额外添加什么，这会增加维护难度，你可以自己fork后添加补丁，自行构建。
